cluster-manifests/kube-system/container-azm-ms-agentconfig.yaml

```diff
@@ -124,26 +124,26 @@ data:
       # When this is enabled (enabled = true), persistent volume metrics including those in the kube-system namespace will be collected
       enabled = true
 
-  alertable-metrics-configuration-settings: |-
+  # alertable-metrics-configuration-settings: |-
```

Comment:
@ferantivero Is there a benefit to keeping this here commented, instead of just removing it? I'm wondering if it becomes more confusing to have it here unused.

Reply:
You are right @johndowns , Im removing these commented out lines. Im discussing with you offline, why this suggested change is actually accepted as the better way to proceed.